system-tests/test/yarn_v3.1.1_pnp_spec.ts

```diff
@@ -1,24 +1,9 @@
-import path from 'path'
 import systemTests from '../lib/system-tests'
-import Fixtures from '../lib/fixtures'
-
-const cypressCli = path.join(__dirname, '../../cli/bin/cypress')
-const projectDir = Fixtures.projectPath('yarn-v3.1.1-pnp')
 
 describe('e2e yarn v3.1.1', () => {
   systemTests.it('can compile plugin and test specs', {
     snapshot: false,
-    command: 'yarn',
     browser: 'electron',
     project: 'yarn-v3.1.1-pnp',
-    onRun: async (run) => {
-      await run({
```

Comment:
this one is a bit bizarre, but the `--dev` mode doesn't work exactly correct here has we are using `ts-node` to register the typescript options, which pollutes the tsconfig with `allowSyntheticDefaultImports` and was causing the test to fail. Not sure why we need dev mode here, but it works fine without it to completely isolate us from `ts-node` in our system tests